CI/install-nnpy.sh

```diff
@@ -2,7 +2,7 @@
 set -e
 git clone https://github.com/nanomsg/nnpy.git
 cd nnpy
-sudo python3 -m pip install cffi
-sudo python3 -m pip install --upgrade cffi
-sudo python3 -m pip install .
+python3 -m pip install cffi
```

Comment:
We should consider deprecating nnpy and replacing it with an equivalent nanomsg library. It's not maintained and the installation can fail on more recent versions.

Reply:
I have created this separate issue to track this idea: https://github.com/p4lang/ptf/issues/215